archimedes/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM archimedes-base
```

Comment:
This powers the `release-test` `docker push` commands.  It builds the final true production image, and thus is similar to the docker-compose but without any locally mounted files.